ios/Podfile

```diff
@@ -37,13 +37,28 @@ target 'ReactNativeProgramm' do
   end
   config = use_native_modules!
 
+  target 'ReactNativeProgramm-dev' do
+    inherit! :complete
+    # Pods for dev
+  end
+
+  target 'ReactNativeProgramm-qa' do
+    inherit! :complete
+    # Pods for qa
+  end
+
+  target 'ReactNativeProgramm-prod' do
+    inherit! :complete
+    # Pods for prod
+  end
+
   # Flags change depending on the env values.
   flags = get_default_flags()
 
   use_react_native!(
     :path => config[:reactNativePath],
     # Hermes is now enabled by default. Disable by setting this flag to false.
-    :hermes_enabled => flags[:hermes_enabled],
+    :hermes_enabled => false,
```

Comment:
These flags also can be tampered with in `%project_root&/ios/.xcode.env` file, if there's no such file you can freely create one and add variable there (just put `hermes_enabled=false`). Also, it's always better to provide a comment why we have to disable some default functions.